app/controllers/settings.js

```diff
@@ -2,16 +2,19 @@ import Ember from 'ember';
 
 export default Ember.Controller.extend({
   envVars: Ember.computed.filterBy('model.envVars', 'isNew', false),
-  cronJobs: Ember.computed.sort('model.cronJobs', function(a, b) {
-    if(a.get('branch.defaultBranch')) {
-      return -1;
-    } else if(b.get('branch.defaultBranch')) {
-      return 1;
-    } else {
-      return a.get('branch.name') > b.get('branch.name');
-    }
-  }),
-  branches: Ember.computed.sort('model.branches', function(a, b) {
+  freeBranches: function() {
+    var cronJobs = this.get('model.cronJobs.jobs');
+    var branches = this.get('model.branches').filter(function(branch) {
+      return branch.get('exists_on_github');
+    });
+    return branches.filter(function(branch) {
+      return ! cronJobs.any(function(cron) {
+        return branch.get('name') === cron.get('branch').get('name');
+      });
```

Comment:
If you wanted to get more ES2015 and Ember-pro with this you could shorten to this:

```
return !cronJobs.any(cron => branch.get('name') === cron.get('branch.name'));
```

Concise and elegant!